plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -113,7 +113,7 @@ def create_container_options
 
     # Mount a cache directory for sharing .m2, .ivy2, .bundler directories between build containers.
     api_version_major, api_version_minor = docker_api_version.scan(/(\d+)\.(\d+)/).flatten.map(&:to_i)
-    if api_version_major == 0 || (api_version_major == 1 && api_version_minor <= 14)
+    if api_version_major.zero? || (api_version_major == 1 && api_version_minor <= 14)
```

Comment:
it's a zero check ... cannot just check `.positive?` since it allows 0 and 1 with a special minor version ...